src/wallet/mod.rs

```diff
@@ -4832,7 +4752,7 @@ pub(crate) mod test {
         let (wallet, _, _) = get_funded_wallet(get_test_tr_repeated_key());
         let addr = wallet.get_address(AddressIndex::New).unwrap();
 
-        let path = vec![("rn4nre9c".to_string(), vec![0])]
+        let path = vec![("e5mmg3xh".to_string(), vec![0])]
```

Comment:
Just curious, why did the policy path id change here? is it related to the miniscript update?